src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendarWidth;
+  gap: $calendarGap;
+  padding-top: $calendarSide;
```

Comment:
not that it impacts anything, but just wondering why would you need it?
without padding it looks more centered.
But of course it's a matter of preference)

Reply:
yes, i don't actually need it but the bottom space looked ugly to me that's why i put it 😁